couchbase/tests/conftest.py

```diff
@@ -0,0 +1,160 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import subprocess
+from time import sleep
+
+import pytest
+import requests
+
+from common import (
+    HERE, HOST, PORT, DATA_PORT, SYSTEM_VITALS_PORT,
+    URL, TAGS, BUCKET_NAME,
+    USER, PASSWORD
+)
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture(scope="session")
```

Comment:
In order to avoid having to make this a test function argument, let's make this `@pytest.fixture(scope="session", autouse=True)`.

Reply:
I believe that will also run the unit tests with the setup fixture, which will slow them down. Is there a way to set `autouse=True` specifically for the integration tests? Either way, I've added a little bit of modularity to the fixtures so I'm not sure if this will be make a meaningful impact to performance.

Reply:
Good point. Let's discuss whether or not separate tests are actually desired on Monday.

Reply:
Sounds good :)